### PR TITLE
Ability for users to join/leave events

### LINKED ISSUE #20: Ability for users to join/leave events
Users should be able to join events that are not full.

Users should be able to leave events before they begin.

### PR DESCRIPTION
**Pull request by bpietraga**
Closes #20

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -10,22 +10,34 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20160725100442) do
+ActiveRecord::Schema.define(version: 20160727124519) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
 
+  create_table "comments", force: :cascade do |t|
```
Comment (hav3loc) [inline db/schema.rb]: We need to delete this for now, my bad.

File: app/controllers/user_event_controller.rb [hunk 2]
```diff
@@ -0,0 +1 @@
+class EventsController < ApplicationController
```
Comment (maciek1995) [inline app/controllers/user_event_controller.rb]: ten plik jest potrzebny?

File: app/controllers/events_controller.rb [hunk 3]
```diff
@@ -18,9 +18,10 @@ def edit
   end
 
   def create
-    @event = current_user.events.new(event_params)
+    @event = current_user.owned_events.new(event_params)
 
     if @event.save
+      @set_owner_of_event = UserEvent.create(user_id: current_user.id, event_id: @event.id, owner: true)
```
Comment (maciek1995) [inline app/controllers/events_controller.rb]: ta zmienna jest gdzies wykorzystywana?

File: app/policies/event_policy.rb [hunk 4]
```diff
@@ -1,26 +1,27 @@
-class EventPolicy
-  attr_reader :user, :event
-
-  def initialize(user, event)
-    @user = user
-    @event = event
-  end
+class EventPolicy < ApplicationPolicy
```
Comment (maciek1995) [inline app/policies/event_policy.rb]: I think you can drop that empty line

File: config/routes.rb [hunk 5]
```diff
@@ -6,4 +6,8 @@
   resources :users
   root to: "static_pages#landing_page"
   resources :events
```
Comment (maciek1995) [inline config/routes.rb]: remove resources :events

**Comment by maciek1995** [comment]
+1

**Pull request merged**